specs/_features/focil/fork-choice.md

```diff
@@ -50,44 +56,53 @@ class Store(object):
     latest_messages: Dict[ValidatorIndex, LatestMessage] = field(default_factory=dict)
     unrealized_justifications: Dict[Root, Checkpoint] = field(default_factory=dict)
     inclusion_lists: Dict[Tuple[Slot, Root], List[InclusionList]] = field(default_factory=dict) # [New in FOCIL]
-    inclusion_list_equivocators: Dict[Tuple[Slot, Root], Set[ValidatorIndex]] = field(default_factory=dict)# [New in FOCIL]
+    inclusion_list_equivocators: Dict[Tuple[Slot, Root], Set[ValidatorIndex]] = field(default_factory=dict) # [New in FOCIL]
 
 
 ### New `on_inclusion_list`
 
 `on_inclusion_list` is called to import `signed_inclusion_list` to the fork choice store.
 ```python
 def on_inclusion_list(
-        store: Store, signed_inclusion_list: SignedInclusionList) -> None:
+        store: Store, 
+        signed_inclusion_list: SignedInclusionList, 
+        inclusion_list_committee: Vector[ValidatorIndex, IL_COMMITTEE_SIZE]]) -> None:
     """
     ``on_inclusion_list`` verify the inclusion list before importing it to fork choice store.
     If there exists more than 1 inclusion list in store with the same slot and validator index, remove the original one.
     """
     message = signed_inclusion_list.message
-    # Verify inclusion list slot is bounded to the current slot
-    assert get_current_slot(store) == message.slot
+    # Verify inclusion list slot is either from the current or previous slot
+    assert get_current_slot(store) in [message.slot, message.slot + 1]
```

Comment:
wouldn't it be message.slot - 1 if it's from either the current or the previous slot?

Reply:
If message is from previous slot, `message.slot + 1 == get_current_slot(store)`.